### ISSUE TITLE
Lab 3 - Απορία για overfit_batch

### ISSUE THREAD

**Issue by NickSpanos55** [slp-ntua/patrec-labs#175]
Καλησπέρα σας. Προσπαθούμε να υλοποιήσουμε την μέθοδο του overfit_batch, παρ'ολ'αυτά, το μοντέλο LSTM δεν κάνει overfit ακόμα και για 1 batch. Θα έπρεπε να βλέπουμε 100% accuracy στα training δεδομένα και ,γενικότερα, πρέπει να βλέπουμε τέτοια χαμηλά scores?(τύπου 15%) Σας ευχαριστούμε πολύ.

**Comment by NickSpanos55** [comment]
Το πρόβλημα με το overfit λύθηκε. Αλλά έχουμε ακόμα απορία για τα scores. Σας ευχαριστούμε.

**Comment by pxaris** [comment]
Καλησπέρα, ναι το απλό LSTM πιθανότατα κινείται σ' αυτά τα επίπεδα. 
Δοκιμάστε να αναπτύξετε Bidirectional-LSTM, να αυξήσετε τα LSTM layers (πχ σε 2 ή 3) και να βάλετε dropout (πχ 0.2).
